tests/runtime/ray/test_client.py

```diff
@@ -22,6 +22,7 @@ def required_kwargs(self):
                 "metadata": {},
                 "catch_exceptions": False,
                 "runtime_env": None,
+                "max_retries": 0,
```

Comment:
**suggestion**: please add an inline comment stating we need this option to workaround issues with duplicated run IDs when a Ray worker crashes. I know it's in the PR & ticket descriptions but a future dev might easily overlook this.

Reply:
There's a comment for w/ with invocation `max_retries`, I added a comment here to reference that.